### PR TITLE
2.2.0 -> 2.2.1

### PR DESCRIPTION
**Pull request by yuhunglin**
* Deprecate yarn in favor of npm
* Linter fixes
* Upgrade poppler simple: 0.8.0 -> 0.10.1 [#9](https://github.com/calipersjs/calipers-pdf/pull/9)
* Deprecate travis in favor of github actions

### CODE REVIEW

**Comment by yuhunglin** [comment]
@tlhunter not sure if you want to expand the `changelog.md` with other details. We should have push perms to npm. Also need to cut a tag for the diff to work.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: "calipers-pdf"
+
+on:
+  push:
+    branches:
+      - '*'
```
Comment (tlhunter) [inline .github/workflows/ci.yml]: I think this should be set to `master`. Doing so will trigger a job run but only when merging to master. Right now with `*` it seems to build with a push to any branch, and it seems to do it redundantly outside of a PR, leading to a double run of tests for every PR:
https://github.com/calipersjs/calipers-pdf/pull/10/checks

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 ## Change Log
 
+**2.2.1** — _December 11, 2019_ — [Diff](https://github.com/calipersjs/calipers-pdf/compare/2.2.0...2.2.1)
```
Comment (tlhunter) [inline CHANGELOG.md]: Was this manually created? Looks like the date is off ;)

**Comment by tlhunter** [comment]
I think the _package-lock.json_ file should be added to _.gitignore_, since this repo represents an npm package and npm packages ignore such lock files.

Including it doesn't necessarily hurt anything, but its presence might make developers think it will have some effect on the final npm install tree.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (yuhunglin) [reply .github/workflows/ci.yml]: 👍 . Good news is that GH actions is free for open source projects so we can be pretty aggressive with triggering the runs, but happy to set this to `master`.

**Review by tlhunter — APPROVED** [review]

**Comment by yuhunglin** [comment]
Still need to:
* cut the release to npm.
* push up the tag so that the changelog will actually link to something relevant.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+jobs:
+  ci:
+    name: Build & Test
+    runs-on: ubuntu-latest
+
+    # Run the same tests suite for node 12, 14, 16
+    strategy:
+      matrix:
+        node-version: [12, 14, 16]
```
Comment (tlhunter) [inline .github/workflows/ci.yml:19]: Nice!

**Comment by tlhunter** [comment]
Looks like 2.2.1 was released:
https://www.npmjs.com/package/calipers-pdf/v/2.2.1

**Pull request merged**